src/Lib/Models/CanvasArtboard.vala

```diff
@@ -132,28 +132,28 @@ public class Akira.Lib.Models.CanvasArtboard : Goo.CanvasItemSimple, Goo.CanvasI
         get_label_extent ();
 
         // Init items list
-        items = new List<Models.CanvasItem> ();
+        items = new Akira.Models.ListModel<Models.CanvasItem> ();
     }
 
     public uint get_items_length () {
-        return this.items.length ();
+        return this.items.get_n_items ();
     }
 
     public void remove_item (Models.CanvasItem item) {
-        items.remove (item);
+        items.remove_item (item);
```

Comment:
this is an `async` method, so it should be called as `items.remove_item.begin (item);`